src/components/StudyForm/StudyForm.tsx

```diff
@@ -87,6 +87,17 @@ const StudyForm: React.FC<StudyFormProps> = ({
       disabled,
     });
 
+  const handlePIAutoFill = () => {
+    if (loggedInUser === null) {
+      return;
+    }
+    setValue("metadata_submission.studyForm.piName", loggedInUser.name);
+    if (loggedInUser.email) {
```

Comment:
Note to self/reviewers (no action needed): This is where the app checks whether the email address of the logged-in user is `null` or `""` (or any other [Falsy value](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)). The auto-fill will only happen if the email address is not such a value.